src/libraries/System.Diagnostics.StackTrace/tests/StackTraceTests.cs

```diff
@@ -300,6 +313,52 @@ public void ToString_NullFrame_ThrowsNullReferenceException()
             Assert.Equal(Environment.NewLine, stackTrace.ToString());
         }
 
+        [Fact]
+        public void ToString_ShowILOffset()
+        {
+            // Normal loading case
+            var asm = Assembly.LoadFrom(SourceTestAssemblyPath);
+            try
+            {
+                asm.GetType("Program").GetMethod("Foo").Invoke(null, null);
+            }
+            catch (Exception e)
+            {
+                Assert.Contains(":token ", e.InnerException.StackTrace);
+            }
+
+            // Assembly.Load(Byte[]) case
+            var inMemBlob = File.ReadAllBytes(SourceTestAssembly2Path);
```

Comment:
It would be nice to not have any more test assemblies than we need to - they all slow down the build a bit.

If you use `RemoteExecutor.Invoke` (see other test projects for examples -- you need `<IncludeRemoteExecutor>true</IncludeRemoteExecutor>` in your project and `using Microsoft.DotNet.RemoteExecutor;` ) then it is easy to run code in a child process, and you can do this once for LoadFrom and once for Load (or one in a child process and one not) both using SourceTestAssemblyPath.

Reply:
Actually, since all you want is an assembly without a pdb, another possibility is to copy some existing test assembly (with a suitable throw method in it) to temp, without its pdb. Eg, src\libraries\System.Runtime\tests\TestLoadAssembly maybe.

Reply:
I'll check and modify like your sugestion

Reply:
I tried to test after changing TC like below.
```
            RemoteExecutor.Invoke((asmPath) =>
            {
                AppContext.SetSwitch("Switch.System.Diagnostics.StackTrace.ShowILOffsets", true);
                var asm = Assembly.LoadFrom(asmPath);
                try
                {
                    asm.GetType("Program").GetMethod("Foo").Invoke(null, null);
                }
                catch (Exception e)
                {
                    Assert.Contains(":token ", e.InnerException.StackTrace);
                }
            }, SourceTestAssemblyPath).Dispose();
```

But, when I ran test with the below command, an error occurs in the RemoteExecutor.Invoke().
```
/tmp/net6/corerun -c /tmp/net6 ./xunit.console.dll System.Diagnostics.StackTrace.Tests.dll
```

Error Message
```
Unhandled exception. System.IO.FileNotFoundException: Could not load file or assembly 'exec'. The system cannot find the file specified.

File name: 'exec'

```

Could you give me a guide how can I run TC with RemoteExecutor?

Reply:
I guess it doesn't work well to running under core run (most of us don't do that). If there is an 'exec' file you could copy it into the folder yourself.

Reply:
Thank you.
I found that I can run TC with "dotnet" instead of "corerun".
I'll update TC soon.